fred2/addvariabledlg.cpp

```diff
@@ -138,7 +138,7 @@ void CAddVariableDlg::OnOK() {
 		//		}
 		//
 		//		m_sexp_var_index = sexp_add_variable(temp_value, temp_name, type);
-		//		this get done for free CDialog::OnOk() UpdateData(TRUE);
+		//		this get done for free CDialog::OnOK() UpdateData(TRUE);
```

Comment:
Just something I noticed, but this has OK capitalized, while the actual function use is `Ok`

Reply:
That's the distinction I mentioned in the PR description.  The `CDialog` function is `OnOK`, with the O and K capitalized.  This is the default MFC function that is invoked in several situations, such as when the user presses Enter.  The FRED code occasionally used `OnOk` and `OnButtonOk` so I've renamed all of them to `OnButtonOk`.

Reply:
Oh right, that all makes sense now, thanks!